components/ContinentScreen/ContinentScreen.jsx

```diff
@@ -5,43 +5,40 @@ import styles from "./ContinentScreen.styles";
 
 
 /**
- * ContinentScreen.
+ * Shows the continent screen
  *
  * @param {object} props
- * @param {object} [props.data={}]
- * @param {boolean} [props.isError=false]
- * @param {boolean} [props.isLoading=false]
- * @param {boolean} [props.isOk=true]
- * @param {object} props.router
+ * @param {object} [props.data]
+ * @param {"LOADING"|"ERROR"|"OK"} [props.responseStatus = "OK"]
+ * @param {object} [props.router]
  * @returns {JSX.Element}
  */
 function ContinentScreen(props) {
 
   const {
-    data = {},
-    isError = false,
-    isLoading = false,
-    isOk = true,
+    data,
+    responseStatus = "OK",
     router,
   } = props;
 
 
   return(
     <BackgroundView>
       <ScrollView style={styles.scrollView}>
-        { isLoading &&
+
+        { responseStatus === "LOADING" &&
```

Comment:
make a RESPONSES const in your useFetchEarth hook so that this usage and the hook can reference the same responses

Reply:
I'll address this when I switch to using @apollo/client